00-RELEASENOTES

```diff
@@ -1,16 +1,146 @@
+* Move prepareClientToWrite out of loop for HGETALL command (#1119)
+* Improved hashing algorithm in luaS_newlstr (#1168)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Do security attack check only when command not found to reduce the critical path. (#1212)
+* Accelerate hash table iterator with prefetching (#1501)
+* Replace dict with new hashtable: hash datatype (#1502)
+* Replace dict with new hashtable for sets datatype (#1176)
+* Replace dict with new hashtable: sorted set datatype (#1427)
+* Trim free space from inline command argument strings to avoid excess memory usage (#1213)
+* Increase the IO_THREADS_MAX_NUM. (#1220)
+* Refactor of ActiveDefrag to reduce latencies (#1242)
+* Integrating fast_float to optionally replace strtod (#1260)
+* Improvements for TLS with I/O threads (#1271)
+* Optimize PFCOUNT, PFMERGE command by SIMD acceleration (#1293)
+* Optimize sdscatrepr by batch processing printable characters (#1342)
+* Optimize ZRANK to avoid path comparisons (#1389)
+* move clientCron onto a separate timer (#1387)
+* client struct: lazy init components and optimize struct layout (#1405)
+* Offload reading the replication stream to IO threads (#1449)
+* skip CRC checksumming during diskless full sync with TLS enabled. (#1479)
+* Accelerate hash table iterator with value prefetching (#1568)
+
+New/Modified configurations
+===========================
+* Add io-threads-do-reads config to deprecated config table to have no effect. (#1138)
+
+Build and Packaging changes
+=================================
+* RDMA builtin support (#1209)
+* Fix Valkey binary build workflow, version support changes. (#1429)
+
+Code improvements
+=================
+* Remove valkey specific changes in jemalloc source code (#1266)
+
+Module Improvements
+===================
+* Use `DEEPBIND` flag when loading external modules using `dlopen` (#1703)
+* Add support for MustObeyClient Module API (#1582)
+* Adds support for scripting engines as Valkey modules (#1277)
+
+Cluster improvements
+====================
+* Do election in order based on failed primary rank to avoid voting conflicts (#1018)
+* Make replica CLUSTER RESET flush async based on lazyfree-lazy-user-flush (#1190)
+* Trigger the election as soon as possible when doing a forced manual failover (#1067)
+* Make manual failover reset the on-going election to promote failover (#1274)
+* Brocast a PONG to all node in cluster when role changed (#1295)
+* Manual failover vote is not limited by two times the node timeout (#1305)
+* Automatic failover vote is not limited by two times the node timeout (#1356)
+
+Behavior Changes
+================
+* Rax size tracking (#688)
+* Take hz into account in activerehashing to avoid CPU spikes (#977)
+* Incr expired_keys if the expiration time is already expired (#1517)
+* Fix replica not able to initiate election in time when epoch fails (#1009)
+* Revert "Decline unsubscribe related command in non-subscribed mode" (#1265)
```

Comment:
This is backported to 8.0.x right? Is it then a new behaviour change in 8.1?

Reply:
I also had thoughts on that. If we backport something to 8.0 but it is in a minor release, minor release can also be AFTER we release a new MAJOR release right? So I did a compare against 8.0.0 and did not account for things be introduced to 8.0.1/2

Reply:
This is not backported as far as I can tell to 8.1.

Reply:
I removed it. did not understand the bottom line, but I think it was introduced to 8.0.2 and which is why this can be removed